proxy/frontend/core/src/main/java/org/apache/shardingsphere/proxy/frontend/ShardingSphereProxy.java

```diff
@@ -62,6 +66,12 @@ public final class ShardingSphereProxy {
     public void start(final int port, final List<String> addresses) {
         try {
             List<ChannelFuture> futures = startInternal(port, addresses);
+            Optional<CDCRule> cdcConfig = ProxyContext.getInstance().getContextManager().getMetaDataContexts().getMetaData().getGlobalRuleMetaData().getRules().stream()
+                    .filter(each -> each instanceof CDCRule).map(each -> (CDCRule) each).findFirst();
+            // TODO cdc is actually independent, but coupled with the proxy code
+            if (cdcConfig.isPresent() && cdcConfig.get().isEnable()) {
+                futures.addAll(new ShardingSphereCDCServer().startAsync(cdcConfig.get().getPort(), addresses));
+            }
```

Comment:
1, It should not be added here, since `CDCServer` is similar as ` ShardingSphereProxy`

2, When will CDCServer's `bossGroup` and `workerGroup` shutdown

Reply:
I changed CDCServer extends Thread and start at Bootstarp.

Reply:
OK. I'll check it again